doc/Architecture.md

```diff
@@ -139,6 +139,10 @@ Business services are always declared using an interface and implemented in a se
 
 This application uses [DynamicData](https://github.com/reactivemarbles/DynamicData) to expose observable lists from business services. These can then be used in the presentation layer to create ViewModels that are automatically disposed when their associated items are removed from the list.
 
+### Update required service
```

Comment:
Here I think you should use the same "domain name" instead of the service name. So I think **Forced Update** would be a better title.